packages/office-addin-debugging/src/stop.ts

```diff
@@ -41,5 +43,26 @@ export async function stopDebugging(manifestPath: string, unregisterCommandLine?
         delete process.env.OfficeAddinDevServerProcessId;
     }
 
+    const processIdReadFromFile = parseNumber(readProcessIdFromFile());
```

Comment:
As recommended before, use functions to read and clear the dev server process Id. I would move any parsing that is required into the function so that it returns "number | undefined".